### PR TITLE
[TT-7057] add info about the default value

### PR DESCRIPTION
**Pull request by letzya**
Ticket https://tyktech.atlassian.net/browse/TT-7057

## Description

Explain what the default value is (and if it's zero)
https://tyk.io/docs/tyk-oss-gateway/configuration/#http_server_optionsread_timeout

## How This Has Been Tested

Haven't tested the script that updated the docs

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test coverage to functionality)
Update comment

## Checklist

- [X] I ensured that the documentation is up to date
- [ ] I explained why this PR updates go.mod in detail with reasoning why it's required
- [ ] I would like a code coverage CI quality gate exception and have explained why

### CODE REVIEW

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4496/merge
Commit:  
Triggered by: pull_request (@letzya)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3633092204)

**Review by tbuchaillot — APPROVED** [review]
LGTM

File: config/config.go [hunk 1]
```diff
@@ -350,10 +350,10 @@ type HttpServerOptionsConfig struct {
 	// No longer used
 	OverrideDefaults bool `json:"-"`
 
-	// User -> Gateway network read timeout
+	// User -> Gateway network read timeout. Not setting this config, or setting this to 0, defaults to 120 seconds
```
Comment (zalbiraw) [inline config/config.go:353]: I would say this should client -> Gateway?

**Review by zalbiraw — CHANGES_REQUESTED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4496/merge
Commit: 64144f676692bc646eb02eb3d4b8fbe3496c9c25 
Triggered by: pull_request (@letzya)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3641998219)

**Review by zalbiraw — APPROVED** [review]
LGTM!

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4496/merge
Commit: 2e09d6ae31a8db248611b75b828d699786929014 
Triggered by: pull_request (@letzya)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3642156462)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4496/merge
Commit: 93a82dd39cb156329aeb6f20fb0b73a004719603 
Triggered by: pull_request (@letzya)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3648952532)

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/heads/master
Commit: 545234c830d5ab5c89b9cabc330122b415094998 [TT-7057] add info about the default value (#4496)


Ticket https://tyktech.atlassian.net/browse/TT-7057

## Description



Explain what the default value is (and if it's zero)

https://tyk.io/docs/tyk-oss-gateway/configuration/#http_server_optionsread_timeout

## Related Issue






## Motivation and Context



## How This Has Been Tested





Haven't tested the script that updated the docs

## Screenshots (if appropriate)

## Types of changes



- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing
functionality to change)
- [ ] Refactoring or add test (improvements in base code or adds test
coverage to functionality)
Update comment

## Checklist





- [X] I ensured that the documentation is up to date
- [ ] I explained why this PR updates go.mod in detail with reasoning
why it's required
- [ ] I would like a code coverage CI quality gate exception and have
explained why
Triggered by: push (@letzya)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/3649277166)

**Pull request merged**